contracts/external/traders/Expiry.sol

```diff
@@ -191,10 +225,7 @@ contract Expiry is
             });
         }
 
-        (
-            uint256 owedMarketId,
-            uint32 maxExpiry
-        ) = parseTradeArgs(data);
+        (uint256 owedMarketId, uint256 maxExpiry) = abi.decode(data, (uint256, uint256));
```

Comment:
should it be `uint32 maxExpiry`...?

Reply:
Either should be fine in this case, but I will change it to `uint32` for consistency